### PR TITLE
add "Processes" section to workspace details

### PR DESCRIPTION
**Pull request by akurinnoy**
Test task is done.

### CODE REVIEW

**Comment by benoitf** [comment]
Hello, thanks for your PR.
In order to handle the PR, the PR needs to be signed as Eclipse rules
you may follow https://github.com/codenvy/che/blob/master/CONTRIBUTING.md

CLA is required
https://www.eclipse.org/legal/CLA.php

if you have any problem, let us know

File: src/app/workspaces/workspace-details/processes/workspace-details-processes.controller.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    promiseProcesses.then(() => {
+      this.updateRunningProcesses();
+    });
+  }
+
+  updateRunningProcesses() {
+    this.runningProcesses = this.codenvyRunner.getRunningProcesses(this.workspaceId);
+    for (var i=0; i<this.runningProcesses.length; i++) {
+      var process = this.runningProcesses[i];
+      process.projectName = process.project.substr(1, process.project.length);
```
Comment (ashumilova) [inline src/app/workspaces/workspace-details/processes/workspace-details-processes.controller.js]: here need to check whether first symbol is slash.

File: src/components/api/codenvy-runner.factory.js [hunk 2]
```diff
@@ -63,6 +68,33 @@ class CodenvyRunner {
     return promise;
   }
 
+  /**
+   * Gets running processes in workspace
+   */
+  fetchRunningProcesses(workspaceId) {
```
Comment (ashumilova) [inline src/components/api/codenvy-runner.factory.js]: Here we can fetch all workspace processes and filter them in controllers by our needs.  
Besides, the variable this.processesPerWorkspaceMap = new Map(); - means all processes, but contains only running.

**Comment by ashumilova** [comment]
In the whole the test task is well done. There are only some minor remarks.

**Pull request closed without merging**